stdlib/_csv.pyi

```diff
@@ -29,9 +31,18 @@ class Dialect:
     lineterminator: str
     quoting: _QuotingType
     strict: bool
-    def __init__(self) -> None: ...
-
-_DialectLike: TypeAlias = str | Dialect | type[Dialect]
+    def __init__(
```

Comment:
Shouldn't this be `__new__`?

Reply:
It is `__new__` for `_csv.Dialect`, but `__init__` for `csv.Dialect`. I feel safer to use `__init__` in both cases.